outbox/shipment-service/src/main/java/io/debezium/examples/outbox/shipment/log/MessageLog.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.examples.outbox.shipment.log;
+
+import java.time.Instant;
+import java.util.UUID;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.transaction.Transactional;
+import javax.transaction.Transactional.TxType;
+
+@ApplicationScoped
+public class MessageLog {
```

Comment:
@gunnarmorling This could be nice use case for ISPN tx cache

Reply:
I had thought about that, too, but decided against it as it adds more complexity again:

* needs persistent cache store so its data is still present after restarts
* needs to be transactional, so that consistency with the app's DB is ensured; in the current approach, e.g. the log record will be rolled back if the business op failed for some reason, just by virtue of using the same DB. Avoiding 2PC is kind of the reason for the entire set-up to begin with, so I think using the DB is preferable.

Reply:
@gunnarmorling I understand, then I propose to add a business table to the database and demo so the benefits are obvious on the first sight.

Reply:
Ok, I can do that.

Reply:
Done.